### PR TITLE
added checkout book feature

### PR DESCRIPTION
**Pull request by srv-max**
Example of post request: http://localhost:8080/api/books/checkout

XML Version:

<?xml version="1.0" encoding="UTF-8"?>
<loans>
    <bookId>1</bookId>>
    <branchId>2</branchId>
    <cardNo>2</cardNo>
</loans>

### CODE REVIEW

File: src/main/java/com/ss/lms/LmsApplication.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 	public static void main(String[] args) {
 		SpringApplication.run(LmsApplication.class, args);
+		System.out.println("Spring running");
```
Comment (AveryGC) [inline src/main/java/com/ss/lms/LmsApplication.java:11]: Did you want to leave this part in??

File: src/main/java/com/ss/lms/controller/BorrowerController.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import com.ss.lms.entity.Loans;
+import com.ss.lms.service.BorrowerService;
+
+@RestController
+public class BorrowerController {
+	@Autowired
+	BorrowerService borrowerService;
+
+	@RequestMapping(path = "/api/books/checkout", method = RequestMethod.POST, headers = {
+			"Accept=application/json,application/xml" }, produces = { "application/json", "application/xml" })
```
Comment (AveryGC) [inline src/main/java/com/ss/lms/controller/BorrowerController.java:29]: Im confused about whats happening in the header.  Are you making it only accept request that say application/json or app/xml in the header??

Or is it whenever a json file is sent the info saying the file is json is added to the header automatically?

**Review by AveryGC — APPROVED** [review]

**Comment by srv-max** [comment]
Made some updates and did checkout book function differently. 

Closing Merge request.

**Review by James-H2o — COMMENTED** [review]
Looks good

**Pull request closed without merging**